mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
aliasing to :core could make a mess.  If you expect a uri and get a core, you're not likely to be happy.  there is already a :url view for *all (in base.rb).  I suppose we could alias to that.